src/internal/events/index.ts

```diff
@@ -82,7 +82,13 @@ export function fireCancelableEvent<T>(
   return event.defaultPrevented;
 }
 
-export function fireKeyboardEvent(handler: CancelableEventHandler<BaseKeyDetail>, reactEvent: React.KeyboardEvent) {
+export function fireKeyboardEvent(
+  handler: CancelableEventHandler<BaseKeyDetail> | undefined,
+  reactEvent: React.KeyboardEvent
+) {
+  if (!handler) {
+    return;
```

Comment:
nice!

Reply:
What's nice here, if the same check already happened  on the line 71 anyway?

Reply:
true, but there are some occurrences of `handler && fireKeyboardEvent(onKeyDown..)` across the codebase. All other `fire.*Event` have this check built-in, so why not align this method too?

Reply:
Making it optional is nice. Duplicating already existing check is not

Reply:
Good catch - I will remove the unneeded if statement 👍